SNAPSHOT_SYSTEM.md

```diff
@@ -0,0 +1,97 @@
+# Two-Tier Snapshot System
+
+## Overview
+
+The benchmarking framework now supports an optimized two-tier snapshot system that significantly improves performance and reduces network overhead when running multiple tests with snapshots.
+
+## Architecture
+
+### Tier 1: Initial Snapshots
+- **Purpose**: Downloaded once at benchmark startup and stored persistently
+- **Location**: Typically `/data/snapshots/initial_<nodeType>_<hash>`
+- **Lifecycle**: Created at benchmark startup, persisted across all tests
+- **Usage**: Serves as the source for per-test copies
+
+### Tier 2: Per-Test Snapshots  
+- **Purpose**: Test-specific copies created from initial snapshots
+- **Location**: Test-specific temporary directories
+- **Lifecycle**: Created before each test, cleaned up after test completion
+- **Usage**: Isolated environment for each test run
+
+## Configuration Format
+
+### New YAML Structure
```

Comment:
I think we can remove this file. Maybe move some of this explanation into `configs/examples/snapshot.yml`